pycheribuild/qemu_utils.py

```diff
@@ -88,14 +88,29 @@ def __init__(self, xtarget: CrossCompileTarget, want_debugger=False):
         else:
             raise ValueError("Unknown target " + str(xtarget))
 
-    def disk_image_args(self, image) -> list:
+    def disk_image_args(self, image: Path, image_format: str) -> list:
+        # Probe the disk image format in case someone has overridden the default image path or format is unspecified
+        if not image.exists():
+            # Either we're pretending or we'll complain elsewhere.
```

Comment:
Hm, seems we don't have a config to hand to assert that we are pretending? Should otherwise have been checked already.

Reply:
Yea, `QemuOptions` isn't a subtype of `Project`.  @arichardson mentioned that other things use the `QemuOptions` type, too, and so this seemed fine?

Reply:
I made it separate so it can be reused by the pexpect test scripts. I guess we could have a config base class that just has pretend?

Reply:
Is it OK to merge as written and defer refactoring? :)